### DISCUSSION TITLE
DAB default values

### DISCUSSION THREAD

**Discussion by meareal** [Azure/data-api-builder#1905] [Q&A]
Good morning. 

I found DB retry policy in this link below, which is great.

https://learn.microsoft.com/en-us/azure/data-api-builder/database-connections

Where can I find a list of all the default values and which ones can be overwritten in the config file, for example, things like:

- Max number of records returned (per request before the nextLink is populated)?
- Max query time (before it times out)? Is this based on the Database being used?
- Max depth of child records?

Thank you very much in advance,

Marcelo.

**Comment by seantleonard** [comment]
Apologies for the delay getting back to you. The behaviors you listed above are not user configurabled.

1. 100 records returned before next link is populated
2. isn't documented and I'll add a workitem for us to get that information updated. 
3. Depth of records for a GraphQL request?

**Comment by meareal** [comment]
Good morning and thanks a lot Sean.

Yes, depth of records for a GraphQL. Is it 5?
